commands/ath.js

```diff
@@ -54,7 +54,7 @@ exports.run = (client, message, args) => {
       }
     }
 
-	if(alchemiter==true && quick == false && client.traitcall.traitCheck(client,charid,"COMPUTER")[1])
+	if(quick == false && client.traitcall.traitCheck(client,charid,"COMPUTER")[1])
```

Comment:
doesn't need to check quick either, if it's true there's no harm setting it to true again.